README.md

```diff
@@ -2,11 +2,858 @@
+compile "io.opentelemetry:opentelemetry-sdk:0.2.0"
+```
+
+#### LoggerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-logging:0.2.0"
+```
+
+#### JaegerExporter
+```gradle
+compile "io.opentelemetry:opentelemetry-exporters-jaeger:0.2.0"
+compile "io.grpc:grpc-protobuf:1.27.2"
+compile "io.grpc:grpc-netty:1.27.2"
+```
+
+### Tracer Configuration
+
+To enable tracing in your OpenTelemetry project configure a TracerBean. This bean will be autowired to controllers to create and propagate spans. If you plan to use a trace exporter remember to also include it in this configuration file. In [section 3](#section-3-instrumentation-using-opentelemetry-contrib-spring-in-progress) we will provide an [annotation](#configtracer) to provide this functionality.
+
+A sample OpenTelemetry configuration using LogExporter is shown below: 
+
+```java
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.SimpleSpansProcessor;
+import io.opentelemetry.trace.Tracer;
+
+import io.opentelemetry.exporters.logging.*;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+
+@Configuration
+@EnableAutoConfiguration(exclude = { DataSourceAutoConfiguration.class, HibernateJpaAutoConfiguration.class })
+public class OtelConfig {
+    private static final tracerName = "otel-example"	
+    @Bean
+    public Tracer otelTracer() throws Exception {
+	final Tracer tracer = OpenTelemetry.getTracerFactory().get(tracerName);
+
+	SpanProcessor logProcessor = SimpleSpansProcessor.newBuilder(new LoggingExporter()).build();
+
+	OpenTelemetrySdk.getTracerFactory().addSpanProcessor(logProcessor);
+
+	return tracer;
+    }
+}
+```
+
+
+The file above configures an OpenTelemetry tracer and a span processor. The span processor builds a Logging Exporter which will output spans to the console. Similarly, one could add another exporter, such as the JaegerExporter, to visualize traces on a different back-end. Similar to how the LoggingExporter is configured, a Jaeger configuration can be added to the OtelConfig class above. 
+
+Sample configuration for a Jaeger Exporter:
+
+```java
+//import io.grpc.ManagedChannelBuilder;
+//import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+
+SpanProcessor jaegerProcessor = SimpleSpansProcessor.newBuilder(JaegerGrpcSpanExporter.newBuilder()
+	.setServiceName("otel_FirstService")
+	.setChannel(ManagedChannelBuilder.forAddress("localhost", 14250).usePlaintext().build()).build())
+	.build();
+OpenTelemetrySdk.getTracerFactory().addSpanProcessor(jaegerProcessor);
+```
+     
+### Project Background
```

Comment:
[Suggest] Move this short description back to where you had the running example, BEFORE any instrumentation. 
If you can, put the whole code for both controllers there, but without anything instrumentation related.

Then, when you have the instrumented code here, indicate that the new lines are in bold or something similar., or to compare to the original.